File: mobile/lib/features/verif/verification_page.dart
import 'package:flutter/material.dart';
import '../../shared/widgets/background.dart';

class VerificationPage extends StatelessWidget {
  const VerificationPage({super.key});

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: SafeArea(
        child: BackgroundColor(
          main: Padding(
            padding: const EdgeInsets.symmetric(
              horizontal: 30,
            ),
            child: Container(
              width: MediaQuery.of(context).size.width,
              height: MediaQuery.of(context).size.height * 0.35,
              decoration: BoxDecoration(
                color: Colors.white,
                borderRadius: BorderRadius.circular(16),
              ),
              padding: const EdgeInsets.symmetric(
                horizontal: 20,
                vertical: 20,
              ),
              child: Column(
                mainAxisAlignment: MainAxisAlignment.center,
                crossAxisAlignment: CrossAxisAlignment.start,
                children: [
                  const FittedBox(
                    child: Text(
                      "Verifikasi Email",
                      style: TextStyle(
                        color: Colors.black,
                        fontWeight: FontWeight.w600,
                        fontSize: 26,
                      ),
                      textAlign: TextAlign.left,
                    ),
                  ),
                  const SizedBox(height: 10),
                  const Flexible(
                    child: Text(
                      "Kami telah mengirimkan email konfirmasi. Silahkan klik tautan konfiramsi dalam email tersebut untuk mengaktifkan akun Anda. Jika Anda tidak menemukan email dari owl@gmail.com, mohon periksa folder Spam dan Junk anda",
                      style: TextStyle(
                        color: Colors.grey,
                        fontWeight: FontWeight.w700,
                        fontSize: 14,
                      ),
                    ),
                  ),
                  const SizedBox(height: 20),
                  InkWell(
                    onTap: () {
                      Navigator.of(context).pushReplacementNamed('/login');
                    },
                    child: Container(
                      width: MediaQuery.of(context).size.width,
                      decoration: BoxDecoration(
                        color: Colors.blue,
                        borderRadius: BorderRadius.circular(8),
                      ),
                      padding: const EdgeInsets.all(6),
                      child: const Text(
                        'Back To Login',
                        textAlign: TextAlign.center,
                        style: TextStyle(
                          color: Colors.white,
                          fontWeight: FontWeight.w600,
                        ),
                      ),
                    ),
                  )
                ],
              ),
            ),
          ),
        ),
      ),
    );
  }
}

File: web/src/Features/Verification/VerificationPage.js
export default function VerificationPage(){

    const urlParams = new URLSearchParams(window.location.search);
    const email = urlParams.get('email');

    return(
        <div className="w-screen h-screen flex justify-center items-center bg-OWL-base p-4">
            <div className="bg-white border-gray-300 px-6 py-10 rounded-xl border-2 shadow-md max-w-md">
                <p className="font-bold text-2xl">Verifikasi Email</p>
                <p className="text-sm mt-2 font-medium text-gray-500">
                    Kami telah mengirimkan email konfirmasi ke {email}. Silahkan klik tautan konfirmasi
                    dalam email tersebut untuk mengaktifkan akun Anda. Jika Anda tidak menemukan email dari owl@gmail.com, 
                    mohon periksa folder Spam dan Junk Anda.
                </p>
            </div>
        </div>
    );
}